web/static/app.css

```
@@ -37,35 +37,88 @@ body {
+  height: 3px;
+  background: var(--accent);
+  content: "";
+}
 
-.search { margin-left: auto; }
+.icon {
+  width: 1.25rem;
+  height: 1.25rem;
+  flex: 0 0 auto;
+  fill: none;
+  stroke: currentColor;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)